.github/workflows/integration-test.yml

```diff
@@ -93,6 +93,7 @@ jobs:
           INTEGRATION_TEST: true
           NODE_ENV: production
           DB_URL_DEFAULT: postgres://postgres:postgres@localhost:5432
+          ADDITIONAL_CONNECTOR_PACKAGES: '@logto/connector-mock-sms,@logto/connector-mock-email'
```

Comment:
nit: `'` is not necessary

Reply:
No. Since `@` is a reserved character in yaml, if we do not add `'`, it will fail.